test/unit/lodash.spec.js

```diff
@@ -0,0 +1,628 @@
+'use strict';
+
+var chai = require('chai');
+chai.use(require('sinon-chai'));
+var expect = chai.expect;
+var _ = require('lodash');
+var datasets = require('../../app/datasets');
+var sinon = require('sinon');
+
+
+describe('Lodash training Util', function() {
+
+    describe('attempt', function() {
+        var params,
+            message,
+            fun,
+            attempt;
+        before(function() {
+            params = datasets.attempt();
+            message = params[0];
+            fun = function(message) {
+                throw new Error(message);
+            };
+            attempt = _.attempt(fun);
+        });
+        it('should verify params', function() {
+            expect(typeof message).to.eql('string');
+            expect(message).to.eql('It is simple error!');
+        });
+        it('should throw error object ', function() {
+            expect(_.isError(attempt)).to.eql(true);
+        });
+    });
+
+    describe('identity', function() {
+        var params;
+        before(function(){
+            params = datasets.identity();
+        });
+        it('should check type', function(){
+            var elem = params[0];
+            expect(elem instanceof Object).to.eql(true);
+        });
+        it('should identity object', function(){
+            var identity = _.identity.apply(_, params);
+            expect(identity).to.have.property('eggs', 2);
+            expect(identity).to.have.property('flour', true);
+        });
+    });
+
+    describe('method', function() {
```

Comment:
This test is too complicated. 
You should do somethink like this
```
_.method('a.b')({a: {b: function() {console.log('CONSOLE')}}})
```
And the check is function in path was called

I can explain it more when we meet